### DISCUSSION TITLE
pgvector update to 0.8.0

### DISCUSSION THREAD

**Discussion by frissonlabs** [supabase/supabase#30256] [Questions]
Not sure where to ask this: when can we expect the latest version of pgvector (0.8.0) to be available? Or where can we track the status of this? The latest version introduces a significant and powerful feature (in my opinion): using HNSW indexes with filters.

**Comment by GaryAustin1** [comment]
They don't publish a roadmap so unless an employee is willing to give inside info no way to know.
Here is where pgvector gets added and does not appear 8.0 has been added yet.
https://github.com/supabase/postgres/pulls?q=is%3Apr+is%3Aclosed+pgvector+
